miniscope_io/logging.py

```diff
@@ -48,27 +49,86 @@ def init_logger(
     if log_dir is None:
         log_dir = config.log_dir
     if level is None:
-        level = config.logs.level_stdout
+        level: LOG_LEVELS = config.logs.level_stdout
     if file_level is None:
-        file_level = config.logs.level_file
+        file_level: LOG_LEVELS = config.logs.level_file
     if log_file_n is None:
         log_file_n = config.logs.file_n
     if log_file_size is None:
         log_file_size = config.logs.file_size
 
+    # set our logger to the minimum of the levels so that it always handles at least that severity
+    # even if one or the other handlers might not.
+    min_level = min([getattr(logging, level), getattr(logging, file_level)])
+
     if not name.startswith("miniscope_io"):
         name = "miniscope_io." + name
```

Comment:
Not really related here, but would it hurt to change this to `mio`? `miniscope_io` is pretty long for my small terminal

Reply:
yep! good idea. will do rn

Reply:
done in: https://github.com/Aharoni-Lab/miniscope-io/pull/73/commits/3624fd8aa9c6cdc98d8058bc9b7053d1c32d7628